assets/src/console/pages/gateway-accounts/CardTypesPage.tsx

```diff
@@ -37,35 +37,34 @@ interface Props {
 }
 
 const getCardTypeBrand = (cardType: CardType) => {
-  if (cardType.type && cardType.label) {
-    let label =
-      cardType.brand === CardTypeBrand.Visa ||
-      cardType.brand === CardTypeBrand.MasterCard
-        ? cardType.label + " " + cardType.type.toLowerCase()
-        : cardType.label;
-    return label;
-  } else {
+  if (!cardType.type || !cardType.label) {
     return "";
+  } else {
```

Comment:
@acodegirl you don't need the else here:

```ts
if (!x) {
  return "";
}
// ...
return ...
```

Reply:
@jonathaningram - Addressed all review comments.